app/scripts/directives/blockEditor.js

```diff
@@ -8,15 +8,27 @@ angular.module('confRegistrationWebApp')
       controller: function ($scope, $modal, modalMessage, uuid, expenseTypesConstants) {
         $scope.activeTab = 'options';
         $scope.visibleRegTypes = {};
+
+        //initializing rule operand value in block object 
+        if (angular.isUndefined($scope.block.content) || $scope.block.content === null || $scope.block.content === '') {
+          $scope.block.content = {
+            ruleoperand: 'OR'
+          };
+        }
+
+        if (angular.isUndefined($scope.block.content.ruleoperand)) {
+          $scope.block.content.ruleoperand = 'OR';
```

Comment:
@OzzieOrca  we fixed this issue by saving the value of information question to $scope.block.content.paragraph